README.md

```diff
@@ -80,6 +80,7 @@ Each new policy should contain:
 * `make generate-website-docs` will generate the markdown files required for the website.
 * `make generate-artifacthub-artifacts` will generate or update the artifact hub packages and associated `artifacthub-pkg.yml` file under `/artifacthub` directory.
 * `make generate-all` will generate all artifacts above.
+* `make validate` will run validation checks on the library repo. Currently it validates directory structure of `website/docs` directory.
```

Comment:
It is covered by the repo CI, but do we want to include `make unit-test`? The `generate-artifacthub-artifacts` step used in include the `cd $(ARTIFACTHUB_SCRIPT_DIR); go test` item which was moved under the new `make unit-test`.